.github/workflows/ci.yml

```
@@ -327,3 +327,20 @@ jobs:
           prerelease: false
           files: |
             package-*/*
+            
+  winget:
+    needs: release
+    name: Publish winget package
+    runs-on: windows-latest
```

```suggestion
  winget-publish:
    needs: release
    if: contains(github.ref, '/tags/v')
    runs-on: windows-latest
```

For consistency with other jobs, I removed `name` (not that it's wrong or anything, it's because I like to be consistent with the other jobs... one of my idiosyncracies).

I also added an `if` so it doesn't deploy the continuous builds which are highly unstable and break often.

I think we should use a unique `WINGET_DEPLOY_TOKEN` because then if the token is leaked (not that I don't trust Microsoft or anything), we know how it happened.

---

Yeah, it's only for security reson, I agree

---

This also needs to be guarded so it only happens on windows 
Something like 
`if: ((runner.os == 'Windows') && (contains(github.ref, 'tags/v')))`

---

This conversation was resolved but no change was made. Reopening.

---

Sorry, forgot to commit